test/fixtures.yml

```diff
@@ -312,6 +312,10 @@ flo2cash_simple:
   password: ANOTHERCREDENTIAL
   account_id: ANOTHERCREDENTIAL
 
+flow:
+  api_key: "iY2dfcSvigHVBW9etXofyijdGCG9a1B1Y3U572HwgEMPc9bbX1iHlItSNBBUJ9vo05cDScCvwegCEAibbqhzwZD11HnI1lJUSLtowJc7iBblIlQ8t1dyxcaYmguXYehB"
```

Comment:
I'm not sure how this should work, I see some gateways have API keys and some don't for remote tests...ideally I'd remove these but then the tests would fail?